components/Footer/Footer.module.scss

```diff
@@ -25,12 +37,17 @@
     row-gap: 1.5rem;
```

Comment:
Possibly change justify-content to space-between to give the icons a little breathing room